docs/docs/routing.md

```diff
@@ -10,9 +10,8 @@ Alternatively, your application may include functionality that cannot be handled
 
 Gatsby makes it easy to programmatically control your pages. Pages can be created in three ways:
 
-- In your site's gatsby-node.js by implementing the API
-  [`createPages`](/docs/node-apis/#createPages)
 - Gatsby core automatically turns React components in `src/pages` into pages
+- In your site's gatsby-node.js by implementing the API [`createPages`](/docs/node-apis/#createPages) (This is an advanced feature)
```

Comment:
Since now the first option is by far more important than the other two, I think it's better to get rid of the list (that kind of implies the options are equivalent), e.g.

> The standard way to define a page in your site is to create new file defining a React component in `src/pages`. Each such file will automatically be turned into one or several pages. 
> 
> For more advanced use cases, it's possible to create pages programmatically using the [`createPages` API](/docs/node-apis/#createPages). This can be done either in your project code or by a plugin.

This has nothing to do with your change but I find the terminology confusing: this document is about "Routing" and the section is called "creating routes" but it only talks about how to create pages. Either the title should be renamed or we should add a sentence explaining "When routing, a route is automatically created for each page" or something. But I'm not sure the page/route distinction is even relevant in this context.